src/collective/cover/tests/test_layout_resize.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
+
+Resource  cover.robot
+Library  Remote  ${PLONE_URL}/RobotRemote
+
+Suite Setup  Open Test Browser
+Suite Teardown  Close all browsers
+
+*** Variables ***
+
+${basic_tile_name} =  "collective.cover.basic"
+${tile_class} =  div.cover-tile
+
+*** Test cases ***
+
+Test Columns-Titles and Resize
+
+    Enable Autologin as  Site Administrator
+    Go to Homepage
+
+    Create Cover  Save-Cover  Test-cover
+    Open Layout Tab
+
+    Add Tile  ${basic_tile_name}
+    Click Element  css=i.resizer
+    Wait until element is visible  id=ui-id-1
+    Drag And Drop By Offset  css=a.ui-slider-handle.ui-state-default.ui-corner-all  -30  0
+    Click Element  css=.ui-dialog-titlebar-close
+
+    Save Cover Layout
+
+    Click Link  link=Compose
```

Comment:
This is not necessary.

Reply:
I do this because many of the problems I have encountered have strange behaviors when saving, changing and returning to the design.

Reply:
Remove to see what Travis will complain about.

Reply:
Remove to see what Travis will complain about.

Reply:
According to me, he complains about other tests, which have nothing to do with mine

Reply:
So no problem removing this keyword.

Reply:
Take a look here.